twitchio/ext/sounds/queuemanager.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+from twitchio.ext import sounds
+
+
+class AudioQueueManager:
+    """
+    Manages a queue of audio files to be played sequentially with optional repeat and pause functionalities.
+
+    Attributes:
+        queue (asyncio.Queue[str]): A queue to hold paths of audio files to be played.
```

Comment:
The docstrings are in the wrong format throughout the file

Reply:
Understood, working on fixing them now, thanks!